src/core/server/saved_objects/service/lib/filter_utils.test.ts

```diff
@@ -21,7 +21,7 @@ import { esKuery } from '../../es_query';
 
 import { validateFilterKueryNode, validateConvertFilterToKueryNode } from './filter_utils';
 
-const mockMappings = {
+export const mockMappings = {
   properties: {
     updatedAt: {
```

Comment:
nit: we use `update_at` for the actual mappings so this could be a bit confusing when reading tests. Maybe we can change this to something completely different like `rootField` or otherwise use `updated_at`